common/resource/resourceImpl.go

```diff
@@ -396,9 +396,6 @@ func (h *Impl) Stop() {
 	h.namespaceCache.Stop()
 	h.membershipMonitor.Stop()
 	h.ringpopChannel.Close()
-	if err := h.grpcListener.Close(); err != nil {
-		h.logger.WithTags(tag.Error(err)).Error("failed to close gRPC listener")
-	}
```

Comment:
It turn out that when we close server it closed underlying listener itself, so this line always generated and error.